broadinstitute/pearl
# [JN-359] kit request UX update

UX update for the kit request page.  The main update is upgrading our table filters to take initial values, so that the table is pre-filtered to be only consented participants who have completed required activities who not already had a kit request.  This also adds a new synthetic participant who has completed required activities but hasn't gotten a kit.

<img width="1243" alt="image" src="https://github.com/broadinstitute/juniper/assets/2800795/9832e646-730e-471c-a460-00db7d41105c">

This also does a drive-by of setting the default sort of the participant list to be by enrollee creation date.

TO TEST:
1. repopulate ourhealth
2. go to `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/kits/eligible`
3. confirm you see the table with filters prepopulated, and one participant "OHKITS" listed
